modules/graphql_views/src/Plugin/Deriver/ViewDeriver.php

```diff
@@ -96,7 +96,7 @@ public function getDerivativeDefinitions($basePluginDefinition) {
         $this->derivatives[$id] = [
           'id' => $id,
           'name' => graphql_core_propcase($name) . 'View',
-          'type' => graphql_core_camelcase($type),
+          'type' => "Entity",
```

Comment:
Not sure if this is a problem. Necessary to fully decouple `graphql_views` from `graphql_content`.
And if we implement #86, there might be missing graphql types. In this case it has to be `Entity` so it can be resolved to `HiddenEntity`.